### DISCUSSION TITLE
Add horizontal repeat direction for row panels

### DISCUSSION THREAD

**Discussion by akshay996** [grafana/grafana#56895] [Dashboards, Panels & Visualizations]
![image](https://user-images.githubusercontent.com/59548002/195658048-bd9acd45-1276-4165-b25f-45d2eaa9cd89.png)

![image](https://user-images.githubusercontent.com/59548002/195657848-796a2b82-bc59-41fc-b96a-bd32b026fc4d.png)

In the row panel, it would be really nice if there is an option repeat all the panels within the row panel to repeat horizontally instead of having the default option just to repeat vertically.

**Use-case**: Like in my dashboard (below), the space on the right is empty and the user would need to scroll vertically to see all the repeated panels.
![image](https://user-images.githubusercontent.com/59548002/195658523-980471d1-95ef-4710-9ab8-aed661d01f09.png)

# **Final Solution (How it would look like if implemented):**
![image](https://user-images.githubusercontent.com/59548002/195659243-38ef9212-9c60-4fa2-a371-5c43d41da01b.png)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
